sanic/sanic.py

```diff
@@ -247,7 +243,7 @@ def converted_response_type(self, response):
     def run(self, host="127.0.0.1", port=8000, debug=False, before_start=None,
             after_start=None, before_stop=None, after_stop=None, sock=None,
             workers=1, loop=None, protocol=HttpProtocol, backlog=100,
-            stop_event=None):
+            stop_event=None, logger=None):
```

Comment:
@awiddersheim in my test it was not necessary to have that boolean flag (as well as the current unit test we have for logging). The example still works as well. Essentially, the configuration to basicConfig gets applied. By delaying the configuration to `run()`, we give the user a time to override it. Maybe it should just be in the `sanic` initialization though? In theory that would also work, though I haven't tested it. Or maybe it should still be added as a property of the app, at least, during runtime? I don't see much reasoning for that, but I could be convinced....